AutomationGui.sc

```diff
@@ -277,7 +282,11 @@ AutomationGui {
         // slider goes in-between
         bslider = bounds.copy;
         if (bounds.width > bounds.height) {
-            bslider.left = bload.left + bload.width;
+			if (automation.hideLoadSave.not) {
+				bslider.left = bload.left + bload.width;
+			} {
+				bslider.left = bload.left -bsave.width;
```

Comment:
Please adjust the position from a GUI element that is shown; bsave is not shown, so it should not be relevant. If I got it correctly, this should be 'bsnap.left + bsnap.width'?